src/main/java/seedu/address/model/person/Address.java

```diff
@@ -30,6 +30,28 @@ public Address(String address) {
         value = address;
     }
 
+    /**
+     * Constructs an empty {@code Address}.
+     * Package-private.
+     */
+    Address() {
+        value = null;
+    }
+
+    /**
+     * Returns an empty {@code Address}.
+     */
+    public static Address createEmpty() {
+        return EmptyAddress.get();
+    }
+
+    /**
+     * Returns false
```

Comment:
Consider expanding the comment for `isEmpty()` to explain that `false` indicates a non-empty `Address`.